WalletWasabi/Blockchain/Transactions/TransactionHistoryBuilder.cs

```diff
@@ -74,6 +75,7 @@ public List<TransactionSummary> BuildHistorySummary()
 						Label = spenderTransaction.Label,
 						TransactionId = spenderTxId,
 						BlockIndex = spenderTransaction.BlockIndex,
+						BlockHash = containingTransaction.BlockHash,
```

Comment:
```suggestion
			BlockHash = spenderTransaction.BlockHash,
```

@yahiheb  this is the correct one, right?

Reply:
Yes.